iis/driver.go

```diff
@@ -87,6 +87,10 @@ var (
 			"username": hclspec.NewAttr("username", "string", false),
 			"password": hclspec.NewAttr("password", "string", false),
 		})),
+		"resource_limit": hclspec.NewBlock("resource_limit", false, hclspec.NewObject(map[string]*hclspec.Spec{
+			"cpu_limit":    hclspec.NewAttr("cpu_limit", "number", true),
+			"memory_limit": hclspec.NewAttr("memory_limit", "number", true),
+		})),
```

Comment:
For this feature, I would prefer we use the `resources` stanza from Nomad and apply said values to IIS rather than configuring it within the driver config stanza.

https://www.nomadproject.io/docs/job-specification/resources

The `resources` stanza is a required setting for a nomad job spec, so we shouldn't have to worry about the value not being set. There might be a bit of conversion needed between nomad's value and types to what IIS is expecting, but this will facilitate a better UX for users who use other drivers and their settings.